example-scripts/voctolight/lib/plugins/rpi_gpio.py

```diff
@@ -0,0 +1,32 @@
+# Plugin to provide a tally light interface for a Raspberry Pi's GPIO
+DO_GPIO = True
+try:
+    import RPi.GPIO as GPIO
+    GPIO.setmode(GPIO.BOARD)
+except ImportError:
+    DO_GPIO = False
+
+__all__ = ['RpiGpio']
+
+class RpiGpio:
+    def __init__(self, config):
+        if not DO_GPIO:
+            raise ValueError('RpiGpio will not work on this platform. Is RPi.GPIO installed?')
+
+        # We don't need an explicit event loop pump
+        self.pump = lambda: None
```

Comment:
Why do we ne need to specify one, then? Better have the caller check if the Plugin needs such a facility.

Reply:
Removed.